image/Make.package

```diff
@@ -96,10 +93,17 @@ msdos:
 		[ -f $(DESTDIR)/$$f ] && mcopy -i $(MSDOS_IMAGE) $(MSDOS_COPYOPTS) $(DESTDIR)$$f ::$$f; \
 	done
 	rm Filelist
-	#mdir -i $(MSDOS_IMAGE) -/ -a ::
-	## FIXME setboot needed to copy fat.bin to FAT sector 1 for .config configuration
-	## setboot hasn't been updated yet for -B<secs>,<heads> and not overwrite FAT BPB
-	## thus package manager builds won't create non-.config based FAT boot images yet
+	# Read boot sector, skip FAT BPB, set ELKS PB sectors/heads and write boot
+ifeq ($(SIZE), 360)
+	setboot $(MSDOS_IMAGE) -F -B9,2 $(TOPDIR)/elkscmd/bootblocks/fat.bin
```

Comment:
Please factorize with $(FD_BOOTSECT)

Reply:
FD_BOOTSECT won't work in this case because FD_BOOTSECT is set by Make.defs to ".../minix.bin". We need fat.bin here. I can rearrange Make.defs and create two FD_BOOTSECTS with different names, but didn't want to get into that without your input. Using the prefix $(MINIX_BOOT) would more properly set the directory to use, but its name is quite confusing given this is the FAT image build section.

Reply:
I am afraid you are confusing `FD_BOOTSECT` and `FD_BOOTBLOCK`...

Reply:
Yes, I'm sorry. There is no FD_BOOTSECT in the source tree. 

FD_BOOTBLOCK won't work in this case because FD_BOOTBLOCK is set by Make.defs to ".../minix.bin". We need fat.bin here. I can rearrange Make.defs and create two FD_BOOTBLOCK with different names, but didn't want to get into that without your input. Using the prefix $(MINIX_BOOT) would more properly set the directory to use, but its name is quite confusing given this is the FAT image build section.

Reply:
I my first comment, I just asked to put the repeated string:
```
$(TOPDIR)/elkscmd/bootblocks/fat.bin
```
into a local FD_BOOTSECT to ease the reading...

Reply:
Got it. Just pushed a cleanup. I had some ideas for cleaning the whole image specification mechanism in the package manager, but wanted to keep this PR clean from that sort of stuff.

Reply:
Thanks !